doc/WebRTC.xml

```diff
@@ -360,7 +360,8 @@
             <para role="param">authorization [string]</para>
             <para role="text">Access token that authorizes the device or client.</para>
             <para role="param">id optional [string]</para>
-            <para role="text">The unique ID of the device or client.</para>
+            <para role="text">The unique ID of the device or client. This parameter is deprecated as
+                device or client identity is encoded in access token acquired out-of-band.</para>

```

Comment:
Isn't it better to mention the identity encoded part in the explanation to the authorization argument.